www/src/components/mdx/code-block/index.tsx

```diff
@@ -1,24 +1,37 @@
 import React from 'react';
-import PropTypes from 'prop-types';
-import Highlight, { defaultProps } from 'prism-react-renderer';
+import Highlight, { defaultProps, Language } from 'prism-react-renderer';
 import ReactCodeBlock from './react-code-block';
 import { previewThemes, classes } from './styles';
 import prismTheme from './prism-theme';
 import styles from './index.module.scss';
 
-const CodeBlock = props => {
-    const { language, shouldRender, theme, children } = props;
+interface PropTypes {
+    children: string;
+    language?: Language;
+    theme?: 'light' | 'dark' | 'white';
+    shouldRender?: boolean;
+}
+
+export default function CodeBlock({
+    children,
+    language,
+    theme = 'white',
+    shouldRender = true,
+}: PropTypes): JSX.Element | null {
+    if (!language) {
+        return null;
+    }
 
     if (language === 'jsx' && shouldRender) {
-        return <ReactCodeBlock {...props} />;
+        return <ReactCodeBlock theme={theme}>{children}</ReactCodeBlock>;
     }
 
-    /* eslint-disable react/no-danger */
     return (
         <div>
-            {language === 'html' && shouldRender && (
+            {(language as Language & 'html') === 'html' && shouldRender && (
```

Comment:
What does this mean?

Reply:
chatted offline. the typecast is necessary because Prism doesn't appear to support html out of the box (although it still seems to work for us)